charts/incubator/nitter/values.yaml

```diff
@@ -0,0 +1,60 @@
+image:
+  repository: tccr.io/truecharts/nitter
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:dde49ae0122fe8da6d00432ec26e596efd36e81e0f14851df6ff40e1c656ddf7
+
+nitter:
+  title: "nitter"
+  hostname: "0.0.0.0:10606"
```

Comment:
No need to be in values. Only in secret. with tpl on port section to match the service port

Reply:
thats for the web section:
```
hostname = "nitter.net"
```

Reply:
Sorry, meant the hostname. This is what binds it to the port/host right?

Reply:
you mean this section?
```
address = "0.0.0.0"
port = 8080
```